JSON_for_IO-Link.yaml

```diff
@@ -264,9 +265,6 @@ paths:
                   value:
                     ethIpv4:
                       - ipConfiguration: DHCP
-                        ipAddress: 192.168.100.5
-                        subnetMask: 255.255.255.0
-                        standardGateway: 192.168.100.1
```

Comment:
I would keep the IP address and so on in the GET endpoint even if dhcp is configured. This way we can read the configuration received from dhcp.

Reply:
These modifications are based on the commit 08e31b53623f71465.

Reply:
I agree with vervoortb's statement.